BUILD

```diff
@@ -94,11 +94,15 @@ kernel(
 		"amd64/bin/date": "bin/date",
```

Comment:
What I would prefer, given how smart bldy is, is to just have the kernel depend on //sys/src/cmd:cmd